packages/shared/src/components/cards/Freeform/FreeformList.tsx

```diff
@@ -91,8 +94,10 @@ export const FreeformList = forwardRef(function SharePostCard(
                 'multi-truncate',
               )}
             >
-              {title}
+              {truncatedTitle}
             </CardTitle>
+
+            {true && <ClickbaitShield post={post} />}
```

Comment:
I assume you forgot to remove this check? 😅

Reply:
`debug: force enable clickbait shield`
I meant to for now, since we don't have any data that would trigger them

Reply:
It'll be reverted once before merging

Reply:
Reverted